CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Added `FileManager.list_files_from_disk`
```

Comment:
Nit: `Added` is redundant. Maybe reword to "- `list_files_from_disk` activity to `FileManager` Tool."